### PR TITLE
Added some content to the index page and an about-us-page

### PR DESCRIPTION
**Pull request by mariushilleke**
This should just be an initial commit. There will come better design, better navigation, etc. Just to start ;).

### CODE REVIEW

**Comment by mariushilleke** [comment]
Oh, just an additional comments:

- I already merged @xeho91 's branch for TAWS-24 into this branch, since I guess it will be merged soon into develop. That helped to use the configurations added in Matt's branch already.
- Switched within the work from the TAWS-2 to TAWS-24 since the first got already resolved and the connected task respectively issue should be more fitting for the commit-content (so it became a new one TAWS-24)

File: source/pages/about-us/about-us-page.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>About Us</title>
+	<link rel="stylesheet" type="text/css" href="./styles/about-us-page.css">
+</head>
+
+<body>
+	<div class="center">
+		<h1>Yep, this is the team behind <a href="../../index.html">Armadillo</a></h1>
+	</div>
+	<div class="team-cards">
```
Comment (lax0248917) [inline source/pages/about-us/about-us-page.html:15]: The home page does a great job of being responsive when dropped to mobile size, however here in the about us page, we see that cards are being cut off in that view. 
image

I know this PR is just to get started adding content to the repo but I wanted to show our Team how we can have constructive Code Reviews!!! :) Great Job Marius and I seriously love that we have code showing up in our repo!

File: source/pages/about-us/styles/about-us-page.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	box-sizing: border-box;
+	margin: 0px;
+	padding: 0px;
+	background-color: white;
+}
+
+body {
+	font-family: "Comic Sans MS", cursive, sans-serif;
+}
+
```
Comment (lax0248917) [inline source/pages/about-us/styles/about-us-page.css:11]: The new responsiveness of the cards at mobile presentation looks great.
One thing I would add though:
`h1 {
  text-align: center;
}`  When you shrink down the window the text stacks and sits on top of the left hand margin. A little center alignment would even this out.

File: source/pages/about-us/about-us-page.html (hunk 1 above)
Comment (mariushilleke) [reply source/pages/about-us/about-us-page.html:15]: @lax0248917 Thank you for feedback! Wanted to fulfill a promise to create some first initial content that can be honed then afterwards but it should have been responsive right away :).
Now we achieve the responsiveness using `@media`.

File: source/pages/about-us/styles/about-us-page.css (hunk 2 above)
Comment (lax0248917) [reply source/pages/about-us/styles/about-us-page.css:11]: This of course is completely unnecessary to change at this time, just practicing and harassing the superstars a little bit 😆  

-🚀

Comment (mariushilleke) [reply source/pages/about-us/styles/about-us-page.css:11]: Yup added that, thank you 😃. I changed the width of the `.card`-class to be `max-width`, so they're not getting overwhelming huge, when there is much space.

File: source/pages/about-us/styles/about-us-page.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
+* {
+	box-sizing: border-box;
+	margin: 0px;
+	padding: 0px;
+	background-color: white;
+}
+
+body {
+	font-family: "Comic Sans MS", cursive, sans-serif;
```
Comment (xeho91) [inline source/pages/about-us/styles/about-us-page.css:9]: Okay, that's it. I'm done. Approval denied. ❌

**Comment by xeho91** [comment]
I really love how you utilized the idea of our project directory structure. ❤️ 
There's no point for me to focus on these details, as I know everything that you've put here is just a spark, to make a better work later on

File: source/pages/about-us/styles/about-us-page.css (hunk 3 above)
Comment (mariushilleke) [reply source/pages/about-us/styles/about-us-page.css:9]: Should have used `fuchsia` as background color 🤔 .

**Pull request merged**